maintainers/maintainer-list.nix

```diff
@@ -3447,6 +3447,12 @@
     githubId = 13279982;
     name = "freezeboy";
   };
+  French-isotope = {
```

Comment:
You set you as maintainer using a lowercase name, however here is the first letter is upper case.